### PR TITLE
Move message consts into the message component

### PR DESCRIPTION
**Pull request by justafish**

### CODE REVIEW

File: src/components/06_wrappers/ErrorBoundary/ErrorBoundary.js [hunk 1]
```diff
@@ -21,9 +20,7 @@ class ErrorBoundary extends React.Component {
 
   render() {
     if (this.state.error) {
-      return (
-        <Message message={this.state.error.toString()} type={MESSAGE_ERROR} />
-      );
+      return <Message message={this.state.error.toString()} />;
```
[the remark this answers is not in the document]
Comment (justafish) [reply src/components/06_wrappers/ErrorBoundary/ErrorBoundary.js]: :+1:
I've updated all usages of messaging to not assume any default

File: src/reducers/application.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default (state = initialState, action) => {
       const messages = [...state.messages];
       messages.push({
         message: action.payload.message,
-        type: action.payload.type,
+        type: action.payload.type || null,
```
[the remark this answers is not in the document]
Comment (justafish) [reply src/reducers/application.js]: :+1:

File: src/components/02_atoms/Message/Message.js [hunk 3]
```diff
@@ -13,7 +12,7 @@ const Message = ({ message, type }) => (
 
 Message.propTypes = {
   message: PropTypes.string.isRequired,
-  type: PropTypes.oneOf([MESSAGE_ERROR, MESSAGE_SUCCESS]).isRequired,
+  type: PropTypes.oneOf([MESSAGE_ERROR, MESSAGE_SUCCESS]),
```
Comment (dawehner) [inline src/components/02_atoms/Message/Message.js]: We maid the type required in every other instance, don't we still want this?

File: src/components/06_wrappers/ErrorBoundary/ErrorBoundary.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dawehner) [reply src/components/06_wrappers/ErrorBoundary/ErrorBoundary.js]: Mh, this one is still left ❓

**Pull request merged**